packages/playground/_stories/main/ComboBox/ComboBox.stories.ts

```diff
@@ -74,30 +68,32 @@ Filters.args = {
 		<ui5-cb-item text="Austria"></ui5-cb-item>
 		<ui5-cb-item text="Bulgaria"></ui5-cb-item>
 		<ui5-cb-item text="Germany"></ui5-cb-item>
-		<ui5-cb-item text="United Kingdom"></ui5-cb-item>
 		<ui5-cb-item text="Kazakhstan"></ui5-cb-item>
+		<ui5-cb-item text="The United Kingdom of Great Britain and Northern Ireland"></ui5-cb-item>
+		<ui5-cb-item text="United Kingdom"></ui5-cb-item>
```

Comment:
can we change it to something not existing above such as Dryanovo or Yablanitsa?